runelite-client/src/main/java/net/runelite/client/plugins/kingdomofmiscellania/KingdomPlugin.java

```diff
@@ -51,6 +64,15 @@
 public class KingdomPlugin extends Plugin
 {
 	private static final ImmutableSet<Integer> KINGDOM_REGION = ImmutableSet.of(10044, 10300);
+	private static final String CHAT_MESSAGE_FORMAT = "Your Kingdom of Miscellania favor is %d%%, and your coffer has %s coins.";
+	private static final int SECONDS_IN_DAY = (int) Duration.of(1, ChronoUnit.DAYS).getSeconds();
+	private static final int MAX_WITHDRAWAL_BASE = 50_000;
+	private static final int MAX_WITHDRAWAL_ROYAL_TROUBLE = 75_000;
+	private static final float FAVOR_DECREMENT_BASE = 0.025f;
+	private static final float FAVOR_DECREMENT_ROYAL_TROUBLE = 0.01f;
+	protected static final int MAX_FAVOR = 127;
+
+	private boolean loggingIn, shouldSendNotification;
```

Comment:
Is `shouldSendNotification` needed? It seems like sending notifications is already fully guarded by `config.shouldSendNotifications()`, `loggingIn`,and `throneOfMiscellaniaCompleted()`.

Reply:
True, createNotification() is only called in one place after checking loggingIn. I’ll change when I get home.

Reply:
I think I originally used `shouldSendNotification` as the only flag, then I created `loggingIn` since I had to wait for the first game tick for `royalTroubleCompleted()` to return accurately, making it obsolete. Anyway, I've removed it.